src/include/pbs_internal.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
 #include "portability.h"
 #include "libutil.h"
 #include "auth.h"
+#include "stdbool.h"
```

Comment:
Nit: Use angular brackets for glibc headers (here and other places), it'll never be found inside pbspro/src/include/